tests/store/tracking/test_rest_store.py

```diff
@@ -473,6 +474,40 @@ def test_databricks_paginate_list_experiments(self):
                 assert experiments[0].name == str(next_page_tokens[idx])
                 assert experiments.token == next_page_tokens[idx]
 
+    def test_search_experiments(self):
+        creds = MlflowHostCreds("https://hello")
+        store = RestStore(lambda: creds)
+
+        with mock_http_request() as mock_http:
+            store.search_experiments()
+            self._verify_requests(
+                mock_http,
+                creds,
+                "experiments/search",
+                "POST",
+                message_to_json(SearchExperiments(view_type=ViewType.ACTIVE_ONLY)),
+            )
+
+        with mock_http_request() as mock_http:
+            store.search_experiments(view_type=ViewType.DELETED_ONLY)
+            self._verify_requests(
+                mock_http,
+                creds,
+                "experiments/search",
+                "POST",
+                message_to_json(SearchExperiments(view_type=ViewType.DELETED_ONLY)),
+            )
+
+        with mock_http_request() as mock_http:
+            store.search_experiments(order_by=["name DESC"])
+            self._verify_requests(
+                mock_http,
+                creds,
+                "experiments/search",
+                "POST",
+                message_to_json(SearchExperiments(view_type=ViewType.DELETED_ONLY)),
```

Comment:
Can we add test to create a real request with backend handler routine run ?

Reply:
Added :)